CNI/romana

```diff
@@ -184,6 +187,52 @@ set_up_pod () {
 
 	# Kubernetes expects valid json on output so no one above should echo anything in stdout
 	echo -n $R
+
+	update_policy "$POD" "$NAMESPACE"
+}
+
+# update_policy discovers romana network policies that affect current pod
+# and notifies romana agent to include new pod into policy
+update_policy () {
```

Comment:
This function doesn't really update the policy, though, doesn't it? It discovers a policy that should be applied to a new pod and then tells the k8s-agent to talk to the romana-agent to apply that policy to the new pod. Right?

That's different.